CHANGELOG.next.asciidoc

```diff
@@ -167,6 +167,8 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Normalize AWS RDS CPU Utilization values before making the metadata API call. {pull}39664[39664]
 - Fix behavior of pagetypeinfo metrics {pull}39985[39985]
 - Fix query logic for temp and non-temp tablespaces in Oracle module. {issue}38051[38051] {pull}39787[39787]
+- Fix for GCP to update the period value to default value(60s) when user specifies a value below the minimum period value in GCP.
```

Comment:
Remove the new line added after the changelog entry please.

```suggestion
- Set GCP metrics config period to the default (60s) when the value is below the minimum allowed period. {pull}40020[40020]
```

Reply:
Done